### PR TITLE
V2Wizard: Repository recommendations (HMS-3249)

### PR DESCRIPTION
**Pull request by regexowl**
This PR is a part of Proactive Assistance - Repository recommendations.

New toggle group was added which allows user to see results from non-added EPEL repository in a case that searched term didn't return any results from added custom and distribution repos.

### CODE REVIEW

**Comment by regexowl** [comment]
Alright soo... this is a baby step for the repo recommendations. To try it just ignore any custom repos on repository step, go directly to packages and search for "gnuplot".

I went around some problems, but there is a thing. The thing is that workarounds work quite nicely until the very last step of dispatching the repo change. 

I've tried to get recommendations for all the non-added repositories, but once a package gets back in the flat response, there is no easy way to figure out which repo it belongs to so the repo can't be added to the store.

The only way to kind of infer the connection seems be making an API call for each of the repos, getting the complete lists of their rpms and then searching for the packages by package names within the lists...

So for now there's only EPEL hard coded in. My question is - should I try to hack in the support for "all repo search" on the front end side? If so, do we want some limit on the number of searched repos? Thinking this might not scale the best.

File: src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx [hunk 1]
```diff
@@ -33,93 +43,53 @@ import {
...
+
   const customRepositories = useAppSelector((state) =>
     selectCustomRepositories(state)
   );
+
+  const { data: epelRepo, isSuccess: isSuccessEpelRepo } =
+    useGetRepositoryQuery({
+      uuid:
+        distribution === 'rhel-8'
+          ? '3012f536-eb7c-4cfc-838d-a0cfd553224c'
```
Comment (lucasgarfield) [inline src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx:71]: This feels like a "magic uuid"... how do we know the appropriate UUIDs?

Comment (regexowl) [reply src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx:71]: Those are queried from the content sources API and also seem to be the reason why the search doesn't work on popular repos that are not added - non-added repos don't have any uuid nor existing_name (although the suggested_name seems to be working for search).

image

I wonder if `searchRpm` goes through the added repos only then. I'm not getting any results with popular non-added repo, but when it's added the results are there even though the url of the repo remains the same.

Comment (regexowl) [reply src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx]: Alright, so I was able to get rid of the uuids which indeed felt a bit like magic values. The repo is now fetched based on url which is much easier to confirm in content sources.

**Review by lucasgarfield — APPROVED** [review]
Love it!

**Pull request merged**